api-spec.yml

```diff
@@ -125,6 +125,12 @@ paths:
       responses:
         200:
           description: Successful response
+  /player/playpause:
```

Comment:
I know the other two are 'pause' and 'resume', but 'playpause' felt more natural to me.

Reply:
Yeah, makes sense.